components/header/Header.vue

```diff
@@ -124,7 +124,7 @@
             <el-input
               v-model="searchQuery"
               type="text"
-              class="input-with-select nav-main-container__search-input"
+              class="nav-main-container__search-input"
```

Comment:
is this change intentional?

Reply:
Yeah, this class doen't do anything. Leftover from when I made it.

Reply:
nice